pkg/container/templates/templates_test.go

```diff
@@ -82,11 +82,12 @@ func TestGetDockerfileTemplate(t *testing.T) {
 				"FROM node:",
 				"npm install --save example-package",
 				"COPY --from=builder --chown=appuser:appgroup /build/node_modules /app/node_modules",
-				"ENTRYPOINT [\"npx\", \"--no-install\", \"example-package\", \"--arg1\", \"--arg2\", \"value\"]",
+				"echo \"exec npx $(echo example-package | sed 's/@[^@]*$//'), \"--arg1\", \"--arg2\", \"value\"\" >> entrypoint.sh",
+				"ENTRYPOINT [\"./entrypoint.sh\"]",
 			},
 			wantMatches: []string{
-				`FROM node:\d+-alpine AS builder`, // Match builder stage
-				`FROM node:\d+-alpine`,            // Match runtime stage
+				`FROM node:22-alpine AS builder`, // Match builder stage
+				`FROM node:22-alpine`,            // Match runtime stage
```

Comment:
I suggest you keep the \d+ regex. This will allow us to migrate to the next major version without modifying the tests.